web-ui/src/main/resources/catalog/components/admin/uiconfig/partials/projectionSwitcher.html

```diff
@@ -3,59 +3,71 @@
 		ng-init="parentIndex=$index">
 		<td>
 			<div class="input-group">
-				<span class="input-group-addon" data-translate="">ui-label</span> <input
```

Comment:
Because the EPSG.io lookup uses the _Code_ field, it makes sense to turn the _Label_ and _Code_ field around: _Code_ first, then _Label_ field.